src/main/java/essenmakanan/command/AddIngredientCommand.java

```diff
@@ -16,10 +18,24 @@ public AddIngredientCommand(String toAdd, IngredientList ingredients) {
 
     @Override
     public void executeCommand() {
-        String ingredientTitle = IngredientParser.parseIngredientTitle(toAdd);
-        Ingredient newIngredient = new Ingredient(ingredientTitle);
-        ingredients.addIngredient(newIngredient);
-        ui.showRecentAddedIngredient(ingredientTitle);
+        String[] allIngredients = toAdd.split("i/");
+
+        for (String ingredient : allIngredients) {
+            if (ingredient.isEmpty()) {
+                continue;
+            }
+
+            ingredient = ingredient.replace("i/", "");
+            Ingredient newIngredient = null;
+            try {
+                newIngredient = IngredientParser.parseIngredient(ingredients, ingredient);
+                ingredients.addIngredient(newIngredient);
+                Ui.printAddIngredientsSuccess(newIngredient.getName());
+            } catch (EssenMakananFormatException e) {
+                e.handleException();
+            }
+        }
+
```

Comment:
very clear code. good that it was implemented in the command class and that the command class calls the parsers.